GitUI/CommandsDialogs/FormStash.cs

```diff
@@ -20,13 +20,6 @@ public sealed partial class FormStash : GitModuleForm
         public bool ManageStashes { get; set; }
         private GitStash? _currentWorkingDirStashItem;
 
-        [Obsolete("For VS designer and translation test only. Do not remove.")]
-        private FormStash()
-        {
-            InitializeComponent();
-            CompleteTheInitialization();
```

Comment:
The removal seems to be OK.

```
        private void CompleteTheInitialization()
        {
            KeyPreview = true;
            View.EscapePressed += () => DialogResult = DialogResult.Cancel;
            splitContainer1.SplitterDistance = DpiUtil.Scale(280);
            InitializeComplete();
        }
```

Reply:
The default constructor is only needed for the Windows Forms designer, which only needs `InitializeComponent()`.